.gitignore

```diff
@@ -1,7 +1,9 @@
 MANIFEST
 build/
 dist/
+.tox/
 .idea/
 __pycache__/
+tmp/
```

Comment:
@o11c It's the directory for my temporary scripts for running tox with my local parameters and options. Not sure they worth sharing.